test/test_quantization.py

```diff
@@ -0,0 +1,352 @@
+        self.checkLinear(myModel.sub2.fc2)
+
+        default_eval_fn(myModel, calib_data)
+
+    def test_nested2(self):
+        myModel = NestedModel()
+        qconfig_dict = {
+            'fc3': default_qconfig,
+            'sub2': default_qconfig
+        }
+        myModel = tq.prepare(myModel, qconfig_dict)
+
+        def checkPrepModules(model, before_calib=False):
+            if before_calib:
+                self.checkObservers(model)
+            self.checkNoPrepModules(model)
+            self.checkNoPrepModules(model.sub1)
+            self.checkNoPrepModules(model.sub1.fc)
+            self.checkNoPrepModules(model.sub1.relu)
+            self.checkNoPrepModules(model.sub2)
+            self.checkHasPrepModules(model.sub2.fc1)
+            self.checkHasPrepModules(model.sub2.fc2)
+            self.checkHasPrepModules(model.fc3)
+
+        checkPrepModules(myModel, True)
+
+        default_eval_fn(myModel, calib_data)
+        tq.convert(myModel)
+
+        checkPrepModules(myModel)
+        self.checkLinear(myModel.sub1.fc)
+        self.assertEqual(type(myModel.sub1.relu), torch.nn.ReLU)
+        self.checkQuantizedLinear(myModel.sub2.fc1)
+        self.checkQuantizedLinear(myModel.sub2.fc2)
+        self.checkQuantizedLinear(myModel.fc3)
+
+        default_eval_fn(myModel, calib_data)
+
+    def test_nested3(self):
+        """
+        More complicated nested test case with fallbacks
+        """
+        myModel = NestedModel()
+        custum_options = {
+            'dtype': torch.quint8,
+            'qscheme': torch.per_tensor_affine
+        }
+        custom_qconfig = QConfig(weight=default_observer(),
+                                 activation=default_observer(**custum_options))
+        qconfig_dict = {
+            'fc3': default_qconfig,
+            'sub2': default_qconfig,
+            'sub2.fc1': custom_qconfig
+        }
+        myModel = tq.prepare(myModel, qconfig_dict)
+
+        def checkPrepModules(model, before_calib=False):
+            if before_calib:
+                self.checkObservers(model)
+            self.checkNoPrepModules(model)
+            self.checkNoPrepModules(model.sub1)
+            self.checkNoPrepModules(model.sub1.fc)
+            self.checkNoPrepModules(model.sub1.relu)
+            self.checkNoPrepModules(model.sub2)
+            self.checkHasPrepModules(model.sub2.fc1)
+            self.checkHasPrepModules(model.sub2.fc2)
+            self.checkHasPrepModules(model.fc3)
+
+        checkPrepModules(myModel, True)
+
+        default_eval_fn(myModel, calib_data)
+        tq.convert(myModel)
+
+        checkPrepModules(myModel)
+        self.checkQuantizedLinear(myModel.sub2.fc1)
+        self.checkQuantizedLinear(myModel.sub2.fc2)
+        self.checkQuantizedLinear(myModel.fc3)
+
+        default_eval_fn(myModel, calib_data)
+
+    def test_quant_wrapper(self):
+        model = WrappedModel()
+
+        tq.propagate_qconfig(model)
```

Comment:
Can replace this test and other tests with call to tq.quantize(), with the suggested change below